### DISCUSSION TITLE
Problems in how we use data/GIS: dogmatism for statistical significance

### DISCUSSION THREAD

**Discussion by anmindi** [albertkun/21S-ASIAAM-191A#24] [Past Thinking Caps]
A fundamental flaw with data usage is the over-reliance on positivism and empiricism: a dogmatic adherence to sometimes abstract rules that can further marginalize minoritized peoples for the sake of upholding "proper science/mathematics." Although numbers cannot exactly lie, the people and methodologies behind data collection _do_ hold their own biases, which then in turn influence how data is measured and portrayed. For example, whenever examining data on health disparities, poverty, incarceration, etc., Asian Americans are hardly ever presented---and when they are, the data is aggregated in such a way that disregards the fact that it is incredibly skewed and neglects the vast diversity and disparities amongst this demographic. (It is even worse when Pacific Islanders get thrown in too in a way that often does not make much sense and leads to erasure). Incomplete data portrayals can then be used to further white supremacist agendas and mask critical civil issues. I have been told many times that it is impossible to disaggregate data for Asian Americans and Pacific Islanders because numbers/sample sizes for certain ethnic groups would be too small to be able to achieve statistical significance. If this is the case, then perhaps we should be questioning and refining the rules/methodologies to better meet our social conditions, rather than willfully disenfranchising small minority populations. Empiricism goes hand in hand with GIS because the end result will only be as accurate as the data that is available.

**Comment by annyrviloria** [comment]
Such an important point. I have worked in research for a long while and just staring my PhD journey and I cannot tell you how many biostatistics lectures I have sat through where the concept of statistical significance is not challenged. It is as if the idea that a made up cutoff for determining the likelihood of something happening *should* make perfect sense. Tufuku Zuberi, a brilliant demographer, writes about the imperialist and racist thought embedded in our collective rationalization of statistics in his book "White logic, white methods". I highly recommend it!

**Comment by skarakas1** [comment]
Great point! Earlier this morning I was listening to an interview with Kim Tran, a diversity, equity, and inclusion (DEI) consultant and author, critiquing the DEI industry. She mentioned that many companies and consultants look at organization-scale results which skew statistics--for example, if a company is 80% white, survey responses are likely to show that a majority of employees find the work environment sufficiently tolerant in what is an essentially meaningless statistic because it is not weighted to represent those who would actually be subject to workplace discrimination. I think it's so important as GIS and data science users to critique our own work and try to see how we can do better, and also to provide transparent explanations of the statistical methods behind visualizations. So often statistics are presented as purely factual representations when, in fact, there are so many decisions and uncertainties throughout the processes behind them.
